frontend/packages/kubevirt-plugin/integration-tests/tests/models/virtualMachine.ts

```diff
@@ -186,6 +188,8 @@ export class VirtualMachine extends BaseVirtualMachine {
         waitForStringInElement(vmView.vmDetailStatus(this.namespace, this.name), VM_STATUS.Running),
         VM_BOOTUP_TIMEOUT_SECS,
       );
```

Comment:
it might be easier to just call waitForStatus here now.

Reply:
Done.

Reply:
can you push the latest changes with this fix?

Reply:
ah, I forgot to do it, pushed the latest changes by now.